src/AppBundle/Factory/Creator.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\AppBundle\Factory;
+
+/**
+ * Interface Creator
+ * @package App\AppBundle\Factory
+ */
+interface Creator
```

Comment:
Skąd taka dziwna nazwa? Factory nie lepiej - przynajmniej wiadomo co to jest i co ma robić.

Reply:
Nie miałem pomysłu na klasę, a Factory mogłoby się trochę zlewać - np. przy namespace `App\AppBundle\Factory\Factory`.